PowerShellGet/PSGet.Resource.psd1

```diff
@@ -70,6 +70,10 @@ ConvertFrom-StringData @'
         PublishModuleLocation=Module '{0}' was found in '{1}'.
         InvalidModuleToPublish=Module '{0}' cannot be published because it does not have a module manifest file. Run New-ModuleManifest -Path <PathName> to create a module manifest with metadata before publishing.         
         MissingRequiredManifestKeys=Module '{0}' cannot be published because it is missing required metadata. Verify that the module manifest specifies Description and Author.
+        
+        InvalidCharactersInPreReleaseString=The PreRelease string contains invalid characters. Please ensure that only characters 'a-zA-Z0-9' and possibly hyphen ('-') at the beginning are in the PreRelease string.

```

Comment:
I see, ok I can change it.  I was using this case because your OneNote spec was using it.  Example:  
```powershell
# Valid PreRelease examples
> Update-ModuleManifest -Path .\ContosoServer.psd1 -PreRelease "-alpha12"
```